extra-utils/swayidle/autobuild/defines

```diff
@@ -0,0 +1,5 @@
+PKGNAME=swayidle
+PKGSEC=x11
+PKGDEP="wayland"
+BUILDDEP="wayland-protocols scdoc fish"
+PKGDES="Idle management daemon for Wayland"
```

Comment:
For Wayland, or for Sway?

Reply:
Compatible with any compositor which implement KDE's idle protocol.